### DISCUSSION TITLE
Accordion in a joomla article not working. Pleas help!

### DISCUSSION THREAD

**Discussion by mahagonny** [twbs/bootstrap#39395] [Q&A]
Granted I'm completely ignorant.
I would like to insert an accordion into a joomla article. I tried to insert the code I found as an example, but when I try the preview the accordion doesn't work.
I can't find explanations that are understandable to me.
I thank anyone who wants to help me.
The theme I use is Cassiopeia

**Comment by julien-deramond** [comment]
Let's see if there are Joomla users in the community who can answer.
It seems pretty specific, have you tried to post something in their community tools too; their forum for instance?

**Comment by mahagonny** [comment]
Thank you.
Yes I have but I can't get satisfactory answers or they are too technical for my abilities
